Vision/PID/README.md

```diff
@@ -0,0 +1,129 @@
+
+---
+page_type: sample
+languages:
+- csharp
+products:
+- azure
+- azure-search
+- azure-cognitive-services
+name: Analyze form sample skill for cognitive search
+urlFragment: azure-analyzeform-sample
+description: This custom skill extracts specific fields from the results of a trained form recognition.
+azureDeploy: https://raw.githubusercontent.com/Azure-Samples/azure-search-power-skills/master/Vision/AnalyzeForm/azuredeploy.json
+---
+
+---
+page_type: sample
+languages:
+- python
+products:
+- azure
+- azure-search
+- azure-container-registry
+- azure-functions
+name: PID 
+urlFragment: azure-pid-drawing-sample
+description: This custom skill extracts specific product/equipment names from PI&D drawings.
+Deploy: Manually [deploy the container image as a Azure function](#deployment)
+---
+
+# PID Skill
+
+P&ID is a graphical representation of the actual process plant using various symbols that represent actual equipment. This skill is designed to extract equipment infomration  from specific instrument symbols in engineering diagrams. The skill uses the X, Y coordinates of text extracted by OCR to generate groupings of text based on proximity, vertical and horizontal seperation and alignment. 
+
+For best results, set the normalized images to the higest resolution. You can also edit the parameters within the skill to change the sensitivity of how the tags are grouped. Additional logic is applied product tags to determine tag boundaries and hypheated text. The skill returns two json elements, a tag array and text array.
+
+![diagram](/images/custom_skill_design.png).
+
+## Requirements
+
+This skill requires Docker to build a container that will be deployed as an Azure function.
+
+## Settings
+
+The default configuration of the skill identified tags or equipment and associated text blocks. Tuning the following parameters allos you to se the sensitivity of grouping of individual text spans into a block.
+
+1. ```maxSegment``` defines the max length of a valid text segment
+2. ```leftAlignSensitivity``` defines the sensitivity of the algorithm in matching text blocks that are left aligned
+3. ```rightAlignSensitivity``` defines the sensitivity of the algorithm in matching text blocks that are right aligned
+4. ```centerAlignSensitivty``` defines the sensitivity of the algorithm in matching text blocks that are center aligned
+
+## Deployment
+
+Follow these steps to buld the container and deploy the skill as an Azure Function.
+
+
+1. Navigate to the diagramskill folder and build the docker container ```docker build -t pidskill .```

```

Comment:
diagramskill -> diagram skill (I hope it's not diagrams kill :D )

Reply:
The folder name is a single token diagramskill

Reply:
haha, makes sense. Maybe back tick it then:
 
```suggestion
1. Navigate to the `diagramskill` folder and build the docker container ```docker build -t pidskill .```

```